pg_search/src/api/operator/text.rs

```diff
@@ -50,6 +50,9 @@ fn text_support_request_simplify(arg: Internal) -> Option<ReturnedNodePointer> {
             T_SupportRequestSimplify,
             arg.unwrap()?.cast_mut_ptr::<pg_sys::Node>()
         )?;
+        if (*srs).root.is_null() {
+            return None;
+        }
```

Comment:
This was the possible segfault.  It's possible (and okay) for `pg_sys::SupportRequestSimplify.root` to be the null pointer.  In that case, there's nuthin' we can do.